app/assets.py

```diff
@@ -13,14 +13,12 @@
 )
 
 vendor_css = Bundle(
-    'vendor/semantic.min.css',
+    'vendor/*.css',
```

Comment:
If you want to include a specific css or js file, those belong in the `static` directory, not the bundled `assets` directory. All the assets get compiled to a single minified script, so being specific about dependencies in this instance is useless.